rpc/filters.go

```diff
@@ -115,11 +116,38 @@ func (f *Filter) pollForBlocks() error {
 	}
 }
 
+func (f *Filter) pollForTransactions() error {
+	for {
+		if f.stopped {
+			return nil
+		}
+
+		txs, err := f.backend.PendingTransactions()
+		if err != nil {
+			return err
+		}
+
+		for _, tx := range txs {
+			f.hashes = append(f.hashes, tx.Hash)
+		}
+
+		<-time.After(1 * time.Second)
```

Comment:
I also noticed the test worked much better without the delay, I think it's too long still

Reply:
yes well, lets go back to it as part of refactoring all this test framework.